app/src/main/java/com/example/dogbook/main/fragments/MapFragment.java

```diff
@@ -132,4 +155,39 @@ private void displayLocation() {
         CameraUpdate cameraUpdate = CameraUpdateFactory.newLatLngZoom(latLng, MAP_FOCUS_ZOOM);
         map.animateCamera(cameraUpdate);
     }
+
+    private void getLocationPosts() {
+        ParseQuery<Post> query = ParseQuery.getQuery(Post.class);
+        query.orderByDescending("createdAt");
+        query.include("author");
+        query.whereExists("location");
```

Comment:
Perhaps we move this out into a class that handles creating this fetch like we talked about before?  So, you can do something like:

```
ParseDataQuery.getLocationPostQuery()
```